include/sgx/crypto/types.h

```diff
@@ -19,3 +27,5 @@ struct FaasmSgxEncryptedMsg
     FaasmSgxIv iv;
     FaasmSgxMac mac;
 };
+
+typedef uint8_t FaasmSgxHashedMsg[SGX_SHA256_HASH_SIZE];
```

Comment:
The reason we use arrays instead of pointers all around is because that's the way the underlying SGX crypto expects most parameters ([see here for `sgx_sha256_hash_t`](https://github.com/intel/linux-sgx/blob/5178e0c26b017a05ec41f6c8edf3cbe30736e2ee/common/inc/sgx_tcrypto.h#L107) definition).

And the reason we use pointers to those structures is to be able to return them from function calls.

I am not comfortable neither with the usage of C-arrays. But if we have to use them, and take pointers to them, I feel having a typedef makes things easier.